dist/monet.js

```diff
@@ -824,7 +835,7 @@
         return Return(a);
     };
     Free.liftF = function(functor) {
-        return Suspend(functor.map(Return));
+        return isFunction(functor) ? Suspend(compose(Return, functor)) : Suspend(functor.map(Return));
```

Comment:
`Function` does not have `.map()` property in JS - so we have to treat it differently :(